`build-rpm` failing: Unable to find image 'securedrop-workstation-dom0-config:latest' locally

## Description

A short summary of the issue.

## Steps to Reproduce

Building from various branches. But the comment denominator is main at ced7d33.

See this failing in:
- https://github.com/freedomofpress/openqa-tests-qubesos/pull/7 - OpenQA test [#159786](https://openqa.qubes-os.org/tests/159786#step/install_workstation/64
- https://github.com/freedomofpress/securedrop-workstation/pull/1373 - OpenQA test [#159783](https://openqa.qubes-os.org/tests/159783#step/install_workstation/74)

## Expected Behavior

Build-rpm succeeds.


## Actual Behavior

```
user@sd-dev:~/securedrop-workstation$ make build-rpm
grep: /etc/fedora-release: No such file or directory
Building SecureDop Workstation RPM...
Script started, output log file is 'build-log/securedrop-workstation-20251118.log'.
[+] Building 0.3s (12/12) FINISHED                                                                                                               docker:default
 => [internal] load build definition from Dockerfile                                                                                                       0.0s
 => => transferring dockerfile: 519B                                                                                                                       0.0s
 => [internal] load metadata for quay.io/fedora/fedora:37                                                                                                  0.2s
 => [internal] load .dockerignore                                                                                                                          0.0s
 => => transferring context: 2B                                                                                                                            0.0s
 => [1/7] FROM quay.io/fedora/fedora:37@sha256:02abb391e65619a9e8fb1c1d43cfb6a27c1b9d164ddd01c9364c2a53c931c2a5                                            0.0s
 => [internal] load build context                                                                                                                          0.0s
 => => transferring context: 169B                                                                                                                          0.0s
 => CACHED [2/7] RUN dnf install -y make                                                                                                                   0.0s
 => CACHED [3/7] COPY Makefile Makefile                                                                                                                    0.0s
 => CACHED [4/7] COPY rpm-build/SPECS rpm-build/SPECS                                                                                                      0.0s
 => CACHED [5/7] RUN make build-deps                                                                                                                       0.0s
 => CACHED [6/7] RUN rm -rf rpm-build                                                                                                                      0.0s
 => CACHED [7/7] RUN if test user != root ; then useradd --no-create-home --home-dir /tmp --uid 1000 user && echo "user ALL=(ALL) NOPASSWD:ALL" >> /etc/s  0.0s
 => exporting to image                                                                                                                                     0.0s
 => => exporting layers                                                                                                                                    0.0s
 => => writing image sha256:f8a9bbd1d71fe54d054353593b2669cb9b66b2549ef15437a6dc0460310a5911                                                               0.0s
 => => naming to docker.io/library/securedrop-workstation-dom0-config37                                                                                    0.0s
Unable to find image 'securedrop-workstation-dom0-config:latest' locally
docker: Error response from daemon: pull access denied for securedrop-workstation-dom0-config, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

Run 'docker run --help' for more information
Script done.
make: *** [Makefile:73: build-rpm] Error 125
```

## Comments

Suggestions to fix, any other relevant information.

--- a/scripts/container.sh
+++ b/scripts/container.sh
@@ -48,7 +48,7 @@ function oci_image() {
 function oci_run() {
     find . \( -name '*.pyc' -o -name __pycache__ \) -delete
 
-    NAME="${1}${SUFFIX}"
+    NAME="${1}${FEDORA_VERSION}${SUFFIX}"
 
     $OCI_BIN run \
            --rm \

